docs/platforms/go/guides/slog/index.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: Slog
+description: "Slog is a structured logging library for Go, introduced in Go 1.21. This guide demonstrates how to integrate slog with Sentry to capture and send logs to Sentry."
+---
+
+For a quick reference, there is a [complete example](https://github.com/getsentry/sentry-go/tree/master/_examples/slog) at the Go SDK source code repository.
```

Comment:
This link currently gives me a 404

Reply:
There is an open PR in the Go-SDK repository (linked in PR description). Once it's merged, the link should work.